google/cloud/logging_v2/gapic/logging_service_v2_client.py

```diff
@@ -477,135 +576,6 @@ def write_log_entries(
             request, retry=retry, timeout=timeout, metadata=metadata
         )
 
-    def list_log_entries(
-        self,
-        resource_names,
-        project_ids=None,
```

Comment:
This mysteriously disappears in the new version of the function (no kwarg or positional equivalent) which is a breaking change. 😢